### PR TITLE
Added auth and other details to the UI deployment guide

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/674
Fixes https://github.com/kubeflow/website/issues/673

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/712)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-712--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/

**Comment by sarahmaddox** [comment]
This PR is ready for review. Adding a hold to allow for multiple reviewers.

/hold

**Comment by sarahmaddox** [comment]
Tech writer review complete, so I'm removing the hold.

/hold cancel

File: content/docs/gke/deploy/deploy-ui.md [hunk 1]
```diff
@@ -9,23 +9,85 @@ deploy Kubeflow on GCP. If you prefer to use the
...
 <img src="/docs/images/kubeflow-deployment.png" 
   alt="Kubeflow deployment UI"
   class="mt-3 mb-3 border border-info rounded">
 
+Follow these steps to open the deployment UI and deploy Kubeflow on GCP:
+
+1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
+  in your web browser. You should see a form like the one in the above
+  screenshot.
+1. Sign in to your browser using a GCP account that has administrator privileges
```
Comment (kunmingg) [inline content/docs/gke/deploy/deploy-ui.md]: How about
that has `project owner role` for your GCP project
?

File: content/docs/gke/deploy/deploy-ui.md [hunk 2]
```diff
@@ -9,23 +9,85 @@ deploy Kubeflow on GCP. If you prefer to use the
...
+1. Sign in to your browser using a GCP account that has administrator privileges 
+  for your GCP project.
+1. Complete the following fields on the form:
+
+    * **Project:** Enter your GCP project name.
+    * **Deployment name:** Enter a short name that you can use to recognize this 
+      deployment of Kubeflow. If you plan to use [Cloud Identity-Aware Proxy
+      (Cloud IAP)](https://cloud.google.com/iap/docs/) for access control (see
+      the next option below), make sure you use the same deployment name 
+      on the deployment UI and when creating the OAuth client ID.
```
Comment (kunmingg) [inline content/docs/gke/deploy/deploy-ui.md]: Should we add a link for `creating the OAuth client ID`?

Comment (sarahmaddox) [reply content/docs/gke/deploy/deploy-ui.md]: Done.

File: content/docs/gke/deploy/deploy-ui.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/gke/deploy/deploy-ui.md]: Done.

**Comment by sarahmaddox** [comment]
Thanks @kunmingg PTAL and LGTM if you're happy with the PR.

**Comment by kunmingg** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**